setup.py

```diff
@@ -5,8 +5,8 @@
 
 
 install_requires = (
-    'djangorestframework>=2.4.3,<3',
-    'incuna_mail>=2.0.0,<3',
+    'djangorestframework>=2.4.4,<3',
+    'incuna_mail>=3.0.0,<3.1.0',
```

Comment:
I suppose we can keep `v2.0` but the difference between the two versions is making `template_name` mandatory.